tutorials/Tutorial_of_QA_for_FAST.ipynb

```diff
@@ -0,0 +1,337 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial of QA_FAST(FMRIB's Automated Segmentation Tool)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "FAST (FMRIB's Automated Segmentation Tool) segments a 3D image of the brain into different tissue types (Grey Matter, White Matter, CSF, etc.), whilst also correcting for spatial intensity variations (also known as bias field or RF inhomogeneities). FAST is based on a hidden Markov random field model and an associated Expectation-Maximization algorithm. The whole process is fully automated and can also produce a bias field-corrected input image and a probabilistic and/or partial volume tissue segmentation. It is robust and reliable, compared to most finite mixture model-based methods, which are sensitive to noise.\n",
+    "more information about FAST, please visit https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FAST\n",
+    "\n",
+    "A quality analysis picture showing the segmentation of a skull-removed brain into white matter, gray matter, and cerebrospinal fluid. Segmentation uses the function of FAST in FSL. You can use the command line below to call the FAST function in FSL for splitting. The file type is nifti image.\n",
+    "\n",
+    "fast -t 1  -n 3 -o basename_for_input_skull-removed brain basename_for_outputs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Here is the code of qa function, in the folder of ndmg/ndmg/stats/qa_fast.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import packages"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "qa_fast depends on matplotlib，numpy, nibabel, scipy and uses functions from ndmg's qa_utils."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import nibabel as nb\n",
+    "from scipy import ndimage\n",
+    "from matplotlib.colors import LinearSegmentedColormap\n",
+    "from ndmg.utils.qa_utils import pad_im"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ndmg.stats.qa_fast import qa_fast_png"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "qa_fast_png function can overlay three image of WM, GM and csf with different color"
```

Comment:
describe how it takes slices at 0.35, 0.51, 0.65 (and what those numbers mean) from sagital, axial, and coronal